lib/resources/postgres.rb

```diff
@@ -41,6 +43,7 @@ def initialize
                   end
 
         @data_dir = File.join('/var/lib/pgsql/', version.to_s, 'data')
+        @service = 'postgresql-' + version if version == '9.4'
```

Comment:
Does this only happen with 9.4, or does it happen with 9.5 and 9.6 as well? Do we perhaps need to change this to:

```
@service = 'postgresql-' + version if version.to_f >= 9.4
```